ui/views.py

```diff
@@ -83,13 +83,16 @@ class CollectionDetail(TemplateView):
     def get_context_data(self, collection_key, **kwargs):  # pylint: disable=arguments-differ
         context = super().get_context_data(**kwargs)
         collection = get_object_or_404(Collection, key=collection_key)
-        if not ui_permissions.has_admin_permission(collection, self.request):
+        if not ui_permissions.has_view_permission(collection, self.request):
             raise PermissionDenied
         video_list = Video.objects.filter(collection=collection)
-        default_settings = default_js_settings(self.request)
         context["collection"] = collection
         context["video_list"] = video_list
-        context["js_settings_json"] = json.dumps(default_settings)
+        context["js_settings_json"] = json.dumps({
```

Comment:
@gsidebo the changes here to `context["js_settings_json"]` in the `CollectionDetail` view are relevant to #137, let me know if you'd prefer to modify this in the PR for that issue instead of this one.

Reply:
This view is going away in #137 so I think it's fine just to leave it.